bundles/org.openhab.binding.radiothermostat/README.md

```diff
@@ -33,15 +26,24 @@ The binding has no configuration options, all configuration is done at Thing lev
+| refresh         | Overrides the refresh interval of the thermostat data. Optional, the default is 2 minutes.                                                                                                                                                                                                                                 |
+| logRefresh      | Overrides the refresh interval of the run-time logs & humidity data. Optional, the default is 10 minutes.                                                                                                                                                                                                                  |
+| isCT80          | Flag to enable additional features only available on the CT80 thermostat. Optional, the default is false.                                                                                                                                                                                                                  |
+| disableLogs     | Disable retrieval of run-time logs from the thermostat. Optional, the default is false.                                                                                                                                                                                                                                    |
+| setpointMode    | Controls temporary or absolute setpoint mode. In "temporary" mode the thermostat will temporarily maintain the given setpoint until the next scheduled setpoint time period. In "absolute" mode the thermostat will ignore its program and maintain the given setpoint indefinitely. Optional, the default is 'Temporary'. |
+| clockSync       | Flag to enable the binding to sync the internal clock on the thermostat to match the openHAB host's system clock. Use if the thermostat is not setup to connect to the manufacturer's cloud server. Sync occurs at binding startup and every hour thereafter. Optional, the default is false.                              |
+
+Some notes:
+
+* The main caveat for using this binding is to keep in mind that the web server in the thermostat is very slow. Do not over load it with excessive amounts of simultaneous commands.
+* When changing the thermostat mode, the current temperature set point is cleared and a refresh of the thermostat data is done to get the new mode's set point.
+* Since retrieving the thermostat's data is the slowest operation, it will take several seconds after changing the mode before the new set point is displayed.
+* The 'Program Mode' command is untested and according to the published API is only available on a CT80 Rev B.
+* Humidity information is available only when using a CT80 thermostat.
+* I have noticed that the humidity reported is very inaccurate (much lower than actual humidity) on my test thermostat.
```

Comment:
If this needs to be mentioned, I would suggest to rewrite into to avoid "I" and "my", as readers will probably wonder "who". Do you know if this is a common problem, or if you might just have a defective thermostat? If this could be a single case of a defective device, it might not be worth mentioning?

Reply:
Agree it is awkward considering the writing style of the README. I will remove it since there is no confirmation of a defect beyond my thermostat.